packages/server/database/migrations/20210330163028-addPriorityScale.ts

```diff
@@ -0,0 +1,49 @@
+import {R} from 'rethinkdb-ts'
+import {PokerCards} from 'parabol-client/types/constEnums'
+import {PALETTE} from 'parabol-client/styles/paletteV3'
+
+const priorityScaleId = 'priorityScale'
+const priorityScale = [
+  {
+    id: priorityScaleId,
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    name: 'Priorities',
+    isStarter: true,
+    sortOrder: 0,
```

Comment:
this should be 3 so it doesn't conflict with fibonacci

Reply:
🦅👀!